### PR TITLE
We have added hooks for form and local storage

### PR DESCRIPTION
**Pull request by leQu**
We added two new hooks to our project, and also started using them in our form and our todolist.

### CODE REVIEW

File: src/components/MultiFieldForm.jsx [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-  const handleChange = (event) => {
-    const name = event.target.name;
-    const value = event.target.value;
-    setFieldData({ ...fieldData, [name]: value });
-  };
-
   const handleSubmit = (event) => {
     event.preventDefault();
-    console.log("Submit", fieldData);
+    console.log("Submit", values);
```
Comment (leQu) [inline src/components/MultiFieldForm.jsx:12]: Sorry I think field data is a bette name for this..

File: src/components/ToDoListComponent.jsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import { useState } from "react";
+// import { useState } from "react";
```
Comment (leQu) [inline src/components/ToDoListComponent.jsx]: you should remove this comment

File: src/components/MultiFieldForm.jsx (hunk 1 above)
Comment (leQu) [reply src/components/MultiFieldForm.jsx:12]: NO I think value is better...

File: src/components/ToDoListComponent.jsx (hunk 2 above)
Comment (leQu) [reply src/components/ToDoListComponent.jsx]: valid, I will update this.

File: src/components/MultiFieldForm.jsx (hunk 1 above)
Comment (Nurhussein9) [reply src/components/MultiFieldForm.jsx:12]: fint

**Review by Nerkolito — APPROVED** [review]
10/10 teacher

**Review by AlexanderFalt — APPROVED** [review]

**Pull request merged**